ckanext/canada/templates/public/base.html

```diff
@@ -1,73 +1,83 @@
+      {% block adobe_analytics_creator %}
+        {{ cr.adobe_analytics_creator() }}
+      {% endblock %}
+      <meta property="dcterms:service" content="OG-GO" />
+      <meta property="dcterms:accessRights" content="{{ h.adobe_analytics_login_required(h.full_current_url()) }}" />
+    {% endif %}
+  {% endblock %}
+{% endblock %}
 
-{% block adobe_analytics %}
-{% if h.adobe_analytics_js() %}
-  <script src="{{ h.adobe_analytics_js() }}"></script>
-{% endif %}
-{%- endblock -%}
+{% block links %}
 
-{% set loop11_key = h.loop11_key() %}
-{% if loop11_key %}
-  <script type="text/javascript">
-    window.loop11_key = '{{ loop11_key }}';
-  </script>
-  <script src="//cdn.loop11.com/my/loop11.js" type="text/javascript" async="async"></script>
-{% endif %}
-{%- endblock -%}
+  {{super()}}
+
+  {% block google_analytics %}
+    {% set ga4_id = h.ga4_id() %}
+    {% if ga4_id %}
+      <!-- Google tag (gtag.js) -->
+      <script async src="https://www.googletagmanager.com/gtag/js?id={{ ga4_id }}"></script>
+      <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', '{{ ga4_id }}');
+      </script>
+    {% endif %}
+  {% endblock %}
+
+  {% block adobe_analytics %}
+    {% if not h.is_registry() and h.adobe_analytics_js() %}
+      <script src="{{ h.adobe_analytics_js() }}"></script>
+    {% endif %}
+  {% endblock %}
+
+  {% set loop11_key = h.loop11_key() %}
+  {% if loop11_key %}
+    <script type="text/javascript">
+      window.loop11_key = '{{ loop11_key }}';
+    </script>
+    <script src="//cdn.loop11.com/my/loop11.js" type="text/javascript" async="async"></script>
+  {% endif %}
+{% endblock %}
 
 {%- block wet_scripts -%}
 {%- endblock -%}
 
-{%- block custom_styles %}
+{%- block custom_styles -%}
   {% snippet 'snippets/wet-boew/headresources.html' %}
   {% asset 'canada_public/css' %}
 {%- endblock -%}
 
 
 {%- block htmltag -%}
-{% set lang = h.lang() %}
-<!--[if lt IE 9]>
-<html class="no-js lt-ie9" lang="{{ lang }}" dir="ltr"><![endif]--> <!--[if gt IE 8]><!-->
-<html class="no-js" lang="{{ lang }}" dir="ltr"> <!--<![endif]-->
+  {%- set lang = h.lang() -%}
+  <!--[if lt IE 9]>
+  <html class="no-js lt-ie9" lang="{{ lang }}" dir="ltr"><![endif]--> <!--[if gt IE 8]><!-->
+  <html class="no-js" lang="{{ lang }}" dir="ltr"> <!--<![endif]-->
 {%- endblock -%}
 
 {%- block title -%}
-  {%- block subtitle_2 %}{% endblock -%}
-  {%- if self.subtitle_2()|trim %} {{ g.template_title_deliminater }} {% endif -%}
-  {%- block subtitle %}{% endblock -%}
-  {%- if self.subtitle()|trim %} {{ g.template_title_deliminater }} {% endif -%}
-  {{ _(g.site_title) }}
+  {%- block subtitle_2 -%}{%- endblock -%}
+  {%- if self.subtitle_2() | trim -%}{{ ' - ' }}{%- endif -%}
+  {%- block subtitle -%}{%- endblock -%}
+  {%- if self.subtitle() | trim -%}{{ ' - ' }}{%- endif -%}
```

Comment:
this should be the same
```suggestion
  {%- if self.subtitle_2() | trim %} - {% endif -%}
  {%- block subtitle -%}{%- endblock -%}
  {%- if self.subtitle() | trim %} - {% endif -%}
```